.travis.yml

```diff
@@ -2,12 +2,19 @@ language: scala
 jdk: oraclejdk7
 sudo: false
 before_install: umask 0022
+cache:
+  directories:
+   - $HOME/.scalding.dep.cache
 script:
   - "echo no op"
+addons:
+  apt:
+    packages:
+    - md5deep
 matrix:
   include:
 #BASE TESTS
```

Comment:
Yeah its wired in as its own test target, we sort of had a free simple target we didn't use. so its in there.